airbyte-cdk/python/airbyte_cdk/models/__init__.py

```diff
@@ -1,2 +1,3 @@
 # generated by generate-protocol-files
 from .airbyte_protocol import *
+from .well_known_types import *
```

Comment:
what is this line for?

Reply:
@sherifnada I'm not sure - this was breaking [here](https://github.com/airbytehq/airbyte/actions/runs/3341292210/jobs/5532371392#step:13:19)

Reply:
I ran it locally to see the diff and this popped up. Wanted to see if fixes the build.

Reply:
looks like master is also breaking: https://github.com/airbytehq/airbyte/actions/runs/3341634147/jobs/5533051500